woocommerce/woocommerce-android
# [Deposit Summary] Use default currency to be show by default

Closes: #10170


### Description

The idea is here is to stop relying on the order of currencies that comes from the backend but to sort it, always showing default currency as the first. The PR also brings a small refactoring

### Testing instructions

* Use a store with multiple currencies
* Notice that the default currency is show first even when you change it on the we
